stroyprombeton/tests/tests_selenium_admin.py

```diff
@@ -256,6 +256,17 @@ def test_tree_redirect_to_table_editor_page(self):
         test_search_value = self.browser.find_element_by_id('search-field').get_attribute('value')
         self.assertTrue(test_search_value)
 
+    def test_table_editor_search(self):
+        """Get first row from table for search test"""
+        first_product_name = self.browser.find_element_by_class_name('jqGrid_name')[0].text
+        search_input = self.browser.find_element_by_id('search_field')
+        search_input.clear()
+        search_input.send_keys(str(first_product_name) + Keys.ENTER)
+        wait(1)
```

Comment:
try not to use const valued waits in tests.
Const waits always provide CI test problems.

Here you can use `stroyprombeton.tests.helpers.BaseSeleniumTestCase#send_keys_and_wait` for this purpose